frontend/src/scenes/onboarding/logic.ts

```diff
@@ -15,7 +17,8 @@ export const signupLogic = kea<signupLogicType>({
     listeners: {
         createAccountSuccess: ({ account }) => {
             if (account) {
-                location.href = '/ingestion'
+                const dest = featureFlagLogic.values.featureFlags['onboarding-2822'] ? '/personalization' : '/ingestion'
```

Comment:
This will cause a merge conflict and won't really work as by this point the user creation has not yet propagated to frontend => is not in sync with this.

Using `location.href` here is desired as it forces a reload.

Let's revert this to avoid a merge conflict, you'll see what I did soon.